packages/navigator/src/components/NavigatorTable.vue

```diff
@@ -69,6 +69,12 @@ import { mapState } from 'vuex'
 export default {
   name: 'NavigatorTable',
   data () {
+    let dataExplorerUrl = window.__INITIAL_STATE__.pluginUrls['dataexplorer']
```

Comment:
where do i add this ?

Reply:
Zie view-navigator.ftl en NavigatorController.java in Molgenis.